albumentations/augmentations/transforms.py

```diff
@@ -1963,11 +1964,10 @@ def apply(self, image, ksize=3, sigma=0, **params):
         return F.gaussian_blur(image, ksize, sigma=sigma)
 
     def get_params(self):
-        ksize = np.random.randint(self.blur_limit[0], self.blur_limit[1] + 1)
-        if ksize != 0 and ksize % 2 != 1:
-            ksize = (ksize + 1) % (self.blur_limit[1] + 1)
-
-        return {"ksize": ksize, "sigma": random.uniform(*self.sigma_limit)}
+        return {
+            "ksize": int(random.choice(np.arange(self.blur_limit[0], self.blur_limit[1] + 1, 2))),
+            "sigma": random.uniform(*self.sigma_limit),
```

Comment:
Here I just unified kernel size selection with other functions.

Reply:
I think it's better to do it in another PR, to be sure that we haven't broken anything.